src/test/scenarios/managed-network/output/src/managed-network/azext_managed_network/generated/_help.py

```diff
@@ -74,6 +75,18 @@
                az managed-network mn delete --name "myManagedNetwork" --resource-group "myResourceGroup"
 """
 
+helps['managed-network mn wait'] = """
+    type: command
+    short-summary: Place the CLI in a waiting state until a condition of the managed-network mn is met.
+    examples:
+      - name: Pause executing next line of CLI script until the managed-network mn is successfully provisioned.
+        text: |-
+               az managed-network mn wait --name "myManagedNetwork" --resource-group "myResourceGroup" updated
+      - name: Pause executing next line of CLI script until the managed-network mn is successfully provisioned.

```

Comment:
The example names are the same now. Can you make the them different?

Reply:
how?

Reply:
change `provisioned` to `created/updated/deleted`? Add @haroldrandom to comment who is familiar with help text conventions.

Reply:
Maybe we can add the created or updated in the name?

Reply:
I changed `provisioned` into `created/updated/deleted` @haroldrandom Can you confirm if this is okay for wait command?

Reply:
what do `provisioned ` mean?

Reply:
provisioningStatus is Success?